game/utils/cycle.hh

```diff
@@ -57,6 +58,11 @@ Cycle<Type>& Cycle<Type>::set(Type value) {
 	return *this;
 }
 
+template<class Type>
+Cycle<Type>& Cycle<Type>::operator=(Type value) {
```

Comment:
Since we have some unittests provided by @twollgam please also create a test for the overload 👍

Reply:
Added and tested the unittests for `Cycle::operator=(Type)` 👍